### PR TITLE
Add gopass as password source support

### PR DESCRIPTION
**Pull request by patrickpichler**
Add support for gopass as password source

#284

### CODE REVIEW

File: jiracli/password.go [hunk 1]
```diff
@@ -43,6 +50,21 @@ func (o *GlobalOptions) GetPass() string {
...
+				orig := os.Getenv("PASSWORD_STORE_DIR")
+				os.Setenv("PASSWORD_STORE_DIR", o.PasswordDirectory.Value)
+				defer os.Setenv("PASSWORD_STORE_DIR", orig)
+			}
+			if bin, err := exec.LookPath("gopass"); err == nil {
+				buf := bytes.NewBufferString("")
+				cmd := exec.Command(bin, "show", "-o", o.keyName())
+				cmd.Stdout = buf
+				cmd.Stderr = buf
+				if err := cmd.Run(); err == nil {
```
Comment (mikepea) [inline jiracli/password.go]: Could you add a failure case too please, panic(err) is fine.

File: jiracli/password.go [hunk 2]
```diff
@@ -43,6 +50,21 @@ func (o *GlobalOptions) GetPass() string {
 			if err != nil {
 				panic(err)
 			}
+		} else if o.PasswordSource.Value == "gopass" {
+			if o.PasswordDirectory.Value != "" {
+				orig := os.Getenv("PASSWORD_STORE_DIR")
+				os.Setenv("PASSWORD_STORE_DIR", o.PasswordDirectory.Value)
+				defer os.Setenv("PASSWORD_STORE_DIR", orig)
+			}
+			if bin, err := exec.LookPath("gopass"); err == nil {
```
Comment (mikepea) [inline jiracli/password.go]: Maybe a warning here for the failure case of gopass not being found? My understanding of the flow is that we'd drop back to the default password behaviour

File: jiracli/password.go (hunk 1 above)
Comment (patrickpichler) [reply jiracli/password.go]: I just noticed, that the pass password-source doesn't have error handling at all. Should i add it there too?

Comment (mikepea) [reply jiracli/password.go]: That'd be lovely, thanks!

Comment (patrickpichler) [reply jiracli/password.go]: should be done

**Comment by mikepea** [comment]
Thanks for the swift responses!

Could you please do a final run through 'go fmt' so that the indentation is correct, and I'll merge.

**Comment by mikepea** [comment]
LGTM

**Pull request merged**